umh.docs.umh.app/content/en/docs/getstarted/managingthesystem.md

```diff
@@ -1,142 +1,247 @@
-+++
-title = "2. Managing the System"
-menuTitle = "2. Managing the System"
-description = "Learn the basics of the Management Console and how to import Node-RED and Grafana flows."
-weight = 2000
-+++
+---
+title: 2. Managing the System
+menuTitle: 2. Managing the System
+description: Learn how to manage your UMH instance with the Management Console.
+weight: 2000
+---
 
-  In this chapter, we will guide you through the installation of the Management 
-  Console and your first local instance of the UMH. We will also explain
-  how the basic components work together, by creating a first Node-RED flow and 
-  a Grafana dashboard. Check out the image below for an 
-  overview:
+In this chapter, you will learn how to monitor, manage and configure your UMH
+instance with the Management Console.
 
-![Untitled](/images/getstarted/managingTheSystem/getStartedUMHSimplifiedpng.png)
+At this stage, you should have already installed the UMH on your device. If you
+have not done so, please follow the steps in the [Installation](/docs/getstarted/installation)
+chapter first.
 
-  The United Manufacturing Hub is a collection of microservices, which work
-  together, to provide a complete solution for your manufacturing needs. From 
-  data collection to contextualization and visualization, or data storage. 
-  For rare cases that require a special feature, you can add it to your UMH
-  stack by creating your own custom microservice.
+## A Few Words About the Communication
 
-## 1. Install the UMH
+Now that you have connected a UMH instance to the Management Console, you might
+be curious about how the Management Console communicates with the instance.
 
+The Management Companion is a microservice that runs in conjunction with the UMH.
```

Comment:
The Management Companion is an essential component of the United Manufacturing Hub (UMH), acting as an agent for each UMH instance. Its main roles are:

Monitoring: It continuously checks the health of the UMH, its operating system, and connected devices like PLCs. If it detects issues or potential failures, it can alert your company's helpdesk system.

Management: The Companion allows users to configure and manage the UMH and its related components, enabling system control from a centralized point.

Security: Each incoming message or request, such as the addition of a new data source, is rigorously checked to ensure that only users with the right permissions can make changes. The system is designed to prevent any direct interaction with your servers to maintain security.

For detailed security information, please visit [insert link to the security page].